nulib/meadow
# Update a potentially flappy JS test

# Summary 
In Github Actions, we had a weird failure of a test which was relying on an async matcher.  This may eliminate that flappiness.

# Specific Changes in this PR
- Update the method of targeting a `select` element in a test
- Update to latest version of React Media Player

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test
1. Run Jest unit tests within the `assets/` directory, and they should all pass.
```
npm run test
```
2. When this branch gets merged, all JS tests pass as well.

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
